desktop_version/src/Graphics.cpp

```diff
@@ -129,6 +131,19 @@ Graphics::Graphics()
     warprect.x = 0, warprect.y = 0, warprect.w = 0, warprect.h = 0;
 }
 
+int Graphics::font_idx(char32_t ch) {
+    if (grphx.im_bfont->h > 128) {
```

Comment:
This part is still a hack - if we have the map in the first place, that’s enough of an indicator to use the special path.